scripts/script.js

```diff
@@ -5,6 +5,7 @@ import { database } from "../firebase/firebase-config.js";
 import { onValue, ref } from "firebase/database";
 import { ref as storageRef, uploadBytesResumable, getDownloadURL} from "https://www.gstatic.com/firebasejs/10.14.0/firebase-storage.js";
```

Comment:
use import from firebase npm package `firebase/storage`

Reply:
This folder was added by mistake